tests/pages/settings/profile-page.ts

```diff
@@ -1,12 +1,19 @@
-import { create, triggerable, visitable } from 'ember-cli-page-object';
+import { clickable, create, triggerable, visitable } from 'ember-cli-page-object';
+import AccountDropdown from 'codecrafters-frontend/tests/pages/components/account-dropdown';
 
 export default create({
+  accountDropdown: AccountDropdown,
```

Comment:
Want to use [`createPage`](https://github.com/codecrafters-io/frontend/blob/master/tests/support/create-page.js)? That includes this by default.